### ISSUE TITLE
Libtool change between 0.13.0 and 0.13.1 can't build shared libs for windows

### ISSUE THREAD

**Issue by tonytheodore** [libass/libass#209]
Hi, 

Attempting to update [libass in MXE](https://github.com/mxe/mxe/blob/master/src/libass.mk) (cross-compile to windows) produces many warnings like:

```
*** Warning: linker path does not have real file for library -lws2_32.
*** I have the capability to make that library automatically link in when
*** you link to this library.  But I can only do this if you have a
*** shared version of the library, which you do not appear to have
*** because I did check the linker path looking for a file starting
*** with libws2_32 and none of the candidates passed a file format test
*** using a file magic. Last file checked: /Users/tonyt/dev/mxe/usr/lib/gcc/i686-w64-mingw32.shared/4.9.3/../../../../i686-w64-mingw32.shared/lib//libws2_32.a
```

[Full log](https://gist.github.com/tonytheodore/3d920d5032a7d0c95f09)

It seems the version of libtool used in the latest release (2.2.4) is not able to create shared libraries for windows where the previous release (with libtool 2.4.2) works fine (as does running `libtoolize && autoreconf -fi`).

Is there a new release planned soon or would it be possible to do a micro release with regenerated configury?

Thanks.

**Comment by wm4** [comment]
Nothing changed about our release process, but 0.13.0 and 0.13.1 were prepared by different persons (using the same checklist).

Are you saying our release is broken on Windows because all the insane autoshit scripts were somehow incorrectly generated? Or generated by a too old set of scripts? In that case I'd say let's not allow these insane scripts to waste more of our time, and just regenerate them yourself as part of your build process.

**Comment by wm4** [comment]
Maybe we'll make a new release anyway. But we'll still use whatever autoconf/libtool scripts are installed on whatever desktop system we use.

**Comment by tonytheodore** [comment]
> Or generated by a too old set of scripts?

Yes, an old version of libtool (2.2.4 was released in 2008), though I'm not sure what distro would be shipping that as even the old Debian stable (Wheezy) has [2.4.2 (from 2011)](https://packages.debian.org/wheezy/libtool).

> and just regenerate them yourself as part of your build process.

We'll probably do this in the short term, but it's just pushing the problem downstream as we have little control over what users have installed on their systems. Autotools causes more problems than actual cross-compile issues and I'd like to avoid using them unless we actually have to patch the underlying input files.

> Maybe we'll make a new release anyway. 

Thanks!

> But we'll still use whatever autoconf/libtool scripts are installed on whatever desktop system we use.

No worries, we have less influence over upstream projects than we do our users ;)

**Comment by wm4** [comment]
Apparently Debian unstable uses libtool 2.4.2. That's on my system, and I made release 0.13.0. I'd do the same for 0.13.2.

**Comment by tonytheodore** [comment]
The previous Debian stable is sufficient, maybe a check that:
```
which {autoconf,automake,libtool}
```
all use the package manager's version would help?

**Comment by astiob** [comment]
I did the 0.13.1 release, and I didn’t realize that I wasn’t using the newest libtool (or that it mattered). Sorry. I do generally have the latest version installed as well (2.4.6 right now), so I’ll make sure I actually use that from now on.

**Comment by astiob** [comment]
0.13.2 has been released now, using libtool 2.4.6. Hopefully this is resolved.
